rotkehlchen/db/updates.py

```diff
@@ -166,6 +172,63 @@ def update_rpc_nodes(self, data: list[dict[str, Any]], version: int) -> None:
             new_default_nodes=new_default_nodes,
         )
 
+    def update_accounting_rules(self, data: list[dict[str, Any]], version: int) -> None:
+        log.info(f'Applying update for accounting rules to v{version}')
+        rules_db = DBAccountingRules(self.user_db)
+        for rule_data in data:
+            try:
+                event_type = HistoryEventType.deserialize(rule_data['event_type'])
```

Comment:
all these deserializations can also raise `DeserializatonError`. Do you want to handle this? Or let it propagate. The lack of a docstring does not let me understand the intentions.